packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -3606,6 +3606,43 @@ export function attach(
     try {
       mostRecentlyInspectedElement = inspectElementRaw(id);
     } catch (error) {
+      // the error name is synced with ReactDebugHooks
+      if (error.name === 'ReactDebugToolsRenderError') {
+        let message = 'Error rendering inspected element.';
```

Comment:
This will swallow the original thing that was thrown if it wasn't an error. Can we just always show the thing in the cause?

Reply:
This will not swallow anything -- it's later overridden by the original error message if it exists (line 3621).

Reply:
Then this message is the fallback, and the thrown object/string is logged via `console.error`

Reply:
Okay, that works! Can we make this error message more descriptive? Something like "React DevTools encountered an error while trying to inspect hooks. This is most likely an error in Component X."